.reek.yml

```diff
@@ -125,5 +125,12 @@ detectors:
     enabled: false
 
 exclude_paths:
+  - bin
   - config
+  - coverage
   - db/migrate
+  - log
+  - public
+  - spec
```

Comment:
Is it ok to exclude always spec files? 🤔

Reply:
TBH I wasn't sure about it. Usually test code is different from app code and doesn't necessarily follow the same standards and best practices. That said, if we were to have custom Ruby classes to deal with things, it would be good that they are maintainable such as any other class in the app folder..

I'll change this, and if we have further evidence in the future that excluding spec is a good idea we can add it back. Does that sound good to you?